speech/google/cloud/speech/client.py

```diff
@@ -67,8 +67,8 @@ def sample(self, content=None, source_uri=None, encoding=None,
                sample_rate=None):
         """Factory: construct Sample to use when making recognize requests.
 
-        :type content: bytes
-        :param content: (Optional) Byte stream of audio.
+        :type content: bytes or file
+        :param content: (Optional) Byte stream of audio or file like object.
```

Comment:
The "steam" in "Byte stream" already sounds like it's a file-like object. Instead: "Bytes object containing audio or file-like object which yields bytes."